spacy_llm/tasks/spancat.py

```diff
@@ -1,78 +1,37 @@
-from collections import defaultdict
 from typing import Any, Callable, Dict, Iterable, List, Optional, Union
 
 from spacy.language import Language
 from spacy.pipeline.spancat import spancat_score
 from spacy.tokens import Doc, Span
 from spacy.training import Example
+from spacy.util import filter_spans
 
 from ..compat import Literal
 from ..registry import registry
-from ..ty import ExamplesConfigType
+from ..ty import RequiredExamplesConfigType
 from ..util import split_labels
-from .span import SpanExample, SpanTask
+from .span import SpanExample, SpanReason, SpanTask
 from .templates import read_template
 
-_DEFAULT_SPANCAT_TEMPLATE_V1 = read_template("spancat")
-_DEFAULT_SPANCAT_TEMPLATE_V2 = read_template("spancat.v2")
+_DEFAULT_SPANCAT_TEMPLATE_V3 = read_template("spans.v3")
```

Comment:
Why not `spancat.v3` to be in line with previous files?

Reply:
The spancat and ner templates were duplicated except the 1 line allowing overlapping spans. So I consolidated them into a single file. I could revert if we want to keep these standard but it's an internal implementation detail so we could always change later if the templates need to be more different than they are now.